extensions/cornerstone/src/hooks/useViewportDisplaySets.ts

```diff
@@ -0,0 +1,95 @@
+import { useMemo } from 'react';
+import { useSystem } from '@ohif/core';
+import {
+  getEnhancedDisplaySets,
+  sortByOverlayable,
+  DERIVED_OVERLAY_MODALITIES,
+} from '../components/ViewportDataOverlaySettingMenu/utils';
+
+const LOW_PRIORITY_MODALITIES = ['SR', 'SEG', 'RTSTRUCT'];
```

Comment:
Can this be defined somewhere externally?  It seems like it is a shared list that gets used several places.